core/citrus-base/src/main/java/com/consol/citrus/functions/Functions.java

```diff
@@ -21,17 +21,7 @@
 import java.util.Collections;
 
 import com.consol.citrus.context.TestContext;
-import com.consol.citrus.functions.core.ChangeDateFunction;
-import com.consol.citrus.functions.core.CurrentDateFunction;
-import com.consol.citrus.functions.core.DecodeBase64Function;
-import com.consol.citrus.functions.core.DigestAuthHeaderFunction;
-import com.consol.citrus.functions.core.EncodeBase64Function;
-import com.consol.citrus.functions.core.RandomNumberFunction;
-import com.consol.citrus.functions.core.RandomStringFunction;
-import com.consol.citrus.functions.core.RandomUUIDFunction;
-import com.consol.citrus.functions.core.ReadFileResourceFunction;
-import com.consol.citrus.functions.core.UrlDecodeFunction;
-import com.consol.citrus.functions.core.UrlEncodeFunction;
+import com.consol.citrus.functions.core.*;
```

Comment:
I personally do prefer exact imports over `*`. but I think we have no general rule for this in citrusframework/citrus.